src/test/java/com/postsquad/scoup/web/user/UserAcceptanceTest.java

```diff
@@ -95,9 +98,62 @@ void signUp(String description, SignUpRequest givenSignUpRequest, User expectedU
         );
     }
 
+    @ParameterizedTest
+    @MethodSource("socialSignUpProvider")
+    @DisplayName("신규 사용자는 소셜 서비스를 통해 회원가입을 할 수 있다")
+    void socialSignUp(String description, SignUpRequest givenSocialSignUpRequest, User expectedUser) {
+        // given
+        String path = "/api/users";
+        RequestSpecification givenRequest = RestAssured.given()
+                                                       .baseUri(BASE_URL)
+                                                       .port(port)
+                                                       .basePath(path)
+                                                       .contentType(ContentType.JSON)
+                                                       .body(givenSocialSignUpRequest);
+
+        // when
+        Response actualResponse = givenRequest.when()
+                                              .log().all(true)
+                                              .post();
+
+        // then
+        actualResponse.then()
+                      .statusCode(HttpStatus.CREATED.value());
+        then(userRepository.findById(actualResponse.body().as(long.class)).orElse(null))
+                .as("회원가입 결과 : %s", description)
+                .usingRecursiveComparison()
+                .ignoringFields(ignoringFieldsForResponseWithId)
+                .isEqualTo(expectedUser);
+    }
+
+    private static Stream<Arguments> socialSignUpProvider() {
```

Comment:
접근 제허자 붙이지 않기로 했던 것 같아요!
```suggestion
    static Stream<Arguments> socialSignUpProvider() {
```
테스트케이스가 많아지면 가독성을 위해서 Provider 분리하는 것도 생각해봐야겠네요

Reply:
```
- 예외적으로 필드 injection을 사용한다.
    - 접근제어자 없이 만든다.
- 테스트 메소드 이외에 필요한 메소드는 적절한(e.g. private) 키워드를 붙여준다.
```
위와 같이 컨벤션을 정했는데 필드 인젝션이랑 테스트 메서드만 접근 제어자를 없애고 그 이외는 private 붙이기로 한 것 아닌가요?

Reply:
아.. 저는 Provider 메소드까지 테스트 메소드라고 생각했어요
테스트코드에서 직접적으로 사용하는 메소드라서요
내일 다시 얘기해볼까요?

Reply:
네 좋아요! 아래 링크에서는 private을 provider에서도 써주는 것 같은데 일반적인 경우는 어떤지 잘 모르겠네요
 https://github.com/eugenp/tutorials/blob/master/testing-modules/junit5-annotations/src/test/java/com/baeldung/junit5/parameterized/StringsUnitTest.java

Reply:
저도 일반적인 경우는 잘 모르겠네요
저는 https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-MethodSource 참고하고 작성했던 것 같아요

Reply:
제가 생각했던 테스트 메소드 이외의 메소드는 추가적인 로직이 필요한 메소드들을 생각했는데 그게 제대로 얘기가 되지 않았던 것 같네요

Reply:
공식문서에서는 접근제어자를 안 썼네요! 저도 Baeldung 의존적 개발을 하지 말고 문서를 더 읽어야겠어요ㅋㅋ

Reply:
이 부분은 새리도 안 붙이시는 것 같아 제가 수정하는걸로 논의를 마무리하면 될 것 같아요